demisto_sdk/commands/common/hook_validations/field_base_validator.py

```diff
@@ -464,14 +464,33 @@ def does_not_have_empty_select_values(self) -> bool:
         Returns:
             (bool): True if selectValues does not have empty values, false if contains empty value.
         """
-        if any(
-            select_value == ""
-            for select_value in (self.current_file.get("selectValues") or [])
-        ):
-            (
-                error_message,
-                error_code,
-            ) = Errors.select_values_cannot_contain_empty_values()
+        error_message, error_code = "", ""
+        if self.current_file.get("type") == "singleSelect":
+            select_values = self.current_file.get("selectValues") or []
+            empty_string_appearance = sum(
+                [1 if select_value == "" else 0 for select_value in select_values]
+            )
```

Comment:
```suggestion
            empty_string_appearance = sum(
                select_value == "" for select_value in select_values
            )
```
https://stackoverflow.com/a/12765840